martin/src/pmtiles/mod.rs

```diff
@@ -145,9 +153,34 @@ impl SourceConfigExtras for PmtConfig {
 
     async fn new_sources_url(&self, id: String, url: Url) -> FileResult<TileInfoSource> {
         match url.scheme() {
-            "s3" => Ok(Box::new(
-                PmtS3Source::new(self.new_cached_source(), id, url).await?,
-            )),
+            "s3" => {
+                let force_path_style = self.force_path_style.unwrap_or_else(|| {
+                    ["1", "true"].contains(
```

Comment:
this should be a function, e.g. `get_env_bool` or similar

Reply:
I added it at the bottom of the file.
I could not quite get `FauxEnv`,`OSEnv` &co. to work.
That would require more elaborate refactoring likely